.github/30-build-installation-issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Build/Installation Issue
+about: Use this template for build/installation issues
+
+---
+
+<em>tag:build_template</em>
```

Comment:
Does this tag do anything in particular?  Is it for tracking purposes?

Reply:
Yes that is for our dashboard to pull the metrics.